docs/_includes/navigation.html

```diff
@@ -0,0 +1,14 @@
+<nav class="navbar navbar-toggleable-md navbar-light bg-faded">
```

Comment:
Just noticed this.. navigation.html is used in the layouts. e.g.: https://github.com/UCF/Athena-Framework/blob/5d02b5ff02fc053f3b53ea37b1b14b2be9d18977/docs/_layouts/default.html#L5